SPECS/libunistring/libunistring.spec

```diff
@@ -1,40 +1,45 @@
-Summary:	GNU Unicode string library
-Name:		libunistring
-Version:	0.9.10
-Release:        3%{?dist}
-License:	LGPLv3+
-Url:		http://www.gnu.org/software/libunistring/
-Source0:	http://ftp.gnu.org/gnu/libunistring/%{name}-%{version}.tar.xz
-%define sha1 libunistring=16dc423d3ebd23f365b0ffe7d584428b427f4bde
-Group:		System Environment/Libraries
+Summary:        GNU Unicode string library
+Name:           libunistring
+Version:        0.9.10
+Release:        4%{?dist}
+License:        LGPLv3+
```

Comment:
Could you also verify the license, since you're here?

Reply:
Sure